content/en/docs/gke/deploy/management-setup.md

```diff
@@ -9,118 +9,220 @@ This guide describes how to setup a management cluster which you will use to dep
 
-For a more detailed explanation of the changes affecting Kubeflow 1.1 on Google Cloud, read [kubeflow/gcp-blueprints #123](https://github.com/kubeflow/gcp-blueprints/issues/123).
+For a more detailed explanation of the drastic changes happened in vKubeflow 1.1 on Google Cloud, read [kubeflow/gcp-blueprints #123](https://github.com/kubeflow/gcp-blueprints/issues/123).
 
 ## Install the required tools
 
-1. Install gcloud components
+1.  Install gcloud components
 
-   ```
-   gcloud components install kpt anthoscli beta
-   gcloud components update
-   ```
+    ```bash
+    gcloud components install kpt anthoscli beta
+    gcloud components update
+    ```
 
-1. Install [Kustomize v3.2.1](https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv3.2.1).
+1.  Install [Kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/).
 
-    **Note:** Kubeflow is not compatible with Kustomize versions above 3.2.1. Read [this GitHub issue](https://github.com/kubeflow/manifests/issues/538) for the latest status.
+    **Note:** Starting from Kubeflow v1.2, we fixed the compatibility problem with Kustomize `v3.2.1+`, so you can now install any Kustomize `v3+`, including the latest Kustomize versions.
+
+1.  Install [yq](https://github.com/mikefarah/yq#install).
 
-1. Install [yq](https://github.com/mikefarah/yq#install).
- 
 ## Setting up the management cluster
 
+1.  Fetch the management blueprint to current directory
 
-1. Fetch the management blueprint
+    ```bash
+    kpt pkg get https://github.com/kubeflow/gcp-blueprints.git/management@v1.2.0 ./
+    ```
 
-   ```
-   kpt pkg get https://github.com/kubeflow/gcp-blueprints.git/management@v1.1.0 ./
-   ```
+1.  Fetch the upstream management package
 
-1. Fetch the upstream manifests
+    ```bash
+    cd ./management
+    make get-pkg
+    ```
 
-   ```
-   cd ./management
-   make get-pkg
-   ```
+    Note, all the instructions below assume your current working directory is `./management`.
 
-1. Open up the **Makefile** at `./management/Makefile` and edit the `set-values` rule to set values for the name, project, and location of your management; when you are done the section should look like
+1.  Use kpt to set values for the name, project, and location of your management cluster:
 
-   ```  
-    set-values: 
-    kpt cfg set ./instance name NAME
-    kpt cfg set ./instance location LOCATION
-    kpt cfg set ./instance gcloud.core.project PROJECT
-  
-    kpt cfg set ./upstream/management name NAME
-    kpt cfg set ./upstream/management location LOCATION
-    kpt cfg set ./upstream/management gcloud.core.project PROJECT
+    ```bash
+    kpt cfg set -R . name ${NAME}
+    kpt cfg set -R . gcloud.core.project ${PROJECT}
+    kpt cfg set -R . location ${LOCATION}
```

Comment:
nit: quote variables

https://google.github.io/styleguide/shellguide.html#variable-expansion

Reply:
Done, thanks for the reminder!